src/ClamavValidator/ClamavValidator.php

```diff
@@ -48,6 +48,28 @@ public function validateClamav($attribute, $value, $parameters)
             return true;
         }
 
+        if(is_array($value)) {
+        	$result = TRUE;    /* Assume all will be well */
```

Comment:
I think better use lower case here`$result = true;` for consistency. Also i think we could remove the comment as well.